### PR TITLE
Created user info

### PR DESCRIPTION
**Pull request by earsenii**

### CODE REVIEW

File: src/components/LoginButton/LoginButton.tsx [hunk 1]
```diff
@@ -16,12 +16,12 @@ export const LoginButton: FC = () => {
...
             withCloseButton: false,
             children: (
-                <div className={classes.modalMain}>
+                <Group noWrap direction="column" align="center" spacing={5} className={classes.modalMain}>
                     <Title order={2}>{t('welcome_to')} Skeetry</Title>
                     <Text color="dimmed" size="sm">
                         {t('description')}
                     </Text>
-                    <Link href="/user/want" passHref>
+                    <Link href="/kive" passHref>
```
Comment (alekseytsvetkov) [inline src/components/LoginButton/LoginButton.tsx:24]: we need this href="/kive"?

File: src/components/Profile/Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+interface IProps {
+    user: IUser;
+}
+
+export const Profile: FC<IProps> = ({user}) => {
+    const {t, i18n} = useTranslation('common');
+    const {classes} = useStyles();
+    const {name, username, location, createdAt, followingCount, followersCount} = user;
+    const currentLocale = i18n.language;
+    return (
```
Comment (alekseytsvetkov) [inline src/components/Profile/Profile.tsx:28]: add empty line between const and return please for better readability

File: src/components/ProfileMenu/ProfileMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React, {FC} from 'react';
+
+import {useTranslation} from 'next-i18next';
+import {Gallery, Message, SearchNormal, Settings} from 'iconsax-react';
+import {Menu} from '@mantine/core';
+
+export const ProfileMenu: FC = () => {
+    const {t} = useTranslation('common');
+    return (
```
Comment (alekseytsvetkov) [inline src/components/ProfileMenu/ProfileMenu.tsx:9]: add empty line here too

File: src/layouts/MainLayout/MainLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+import {useStyles} from './styles';
+
+interface IProps {
+    children: ReactNode;
+}
+
+export const MainLayout: FC<IProps> = ({children}: IProps) => {
+    const {classes} = useStyles();
+    return (
```
Comment (alekseytsvetkov) [inline src/layouts/MainLayout/MainLayout.tsx:14]: empty line

File: src/layouts/MainLayout/MainLayout.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        <AppShell fixed header={<HeaderHome />}>
+            <MediaQuery largerThan="md" styles={{width: '40vw'}}>
```
Comment (alekseytsvetkov) [inline src/layouts/MainLayout/MainLayout.tsx:16]: inline style to class

File: src/pages/[username]/want.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+import {MOCK_USERS} from '_mocks';
+import {MainLayout} from '_layouts/MainLayout';
+import {Profile} from '_components/Profile';
+import {Text} from '@mantine/core';
+
+const Want: FC = () => {
+    const router = useRouter();
+    const {username} = router.query;
+    const user = MOCK_USERS.find((u) => u.username === username);
+    if (!user) {
```
Comment (alekseytsvetkov) [inline src/pages/[username]/want.tsx:15]: add empty line

File: src/pages/[username]/want.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        return (
+            <MainLayout>
+                <Text>Тут будет ошибка</Text>
+            </MainLayout>
+        );
+    }
+    return (
```
Comment (alekseytsvetkov) [inline src/pages/[username]/want.tsx:22]: add empty line

File: src/components/HeaderHome/HeaderHome.tsx [hunk 8]
```diff
@@ -4,31 +4,33 @@ import Link from 'next/link';
...
 import {ColorSchemeToggle} from '_components/ColorSchemeToggle';
-import {Header, Text} from '@mantine/core';
+import {Group, Header, MediaQuery, Text} from '@mantine/core';
 
 import {useStyles} from './styles';
 
 export const HeaderHome: FC = () => {
     const {t} = useTranslation('common');
     const {classes} = useStyles();
     return (
```
Comment (alekseytsvetkov) [inline src/components/HeaderHome/HeaderHome.tsx:14]: add empty line

**Review by alekseytsvetkov — CHANGES_REQUESTED** [review]

File: src/layouts/MainLayout/MainLayout.tsx (hunk 5 above)
Comment (earsenii) [reply src/layouts/MainLayout/MainLayout.tsx:16]: @alekseytsvetkov, styles property applied to child when breakpoint matches, with className propety it doesn't work

**Review by alekseytsvetkov — APPROVED** [review]

**Pull request merged**